extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/RegisterShutdownHookPluginTest.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class RegisterShutdownHookPluginTest {
+
+    @Inject
+    Scheduler quartzScheduler;
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset(
+                                    "quarkus.quartz.plugin.shutdownhook.class=org.quartz.plugins.management.ShutdownHookPlugin\n"
```

Comment:
That's not a good plugin to test. We do shutdown the quartz scheduler when a quarkus app/test stops ;-).

Reply:
This test is because if I register an invalid plugin, the Quartz won't start. The test can check if parsing the config to Quartz properties is working fine.